src/main/webapp/controllers/tutorialController.js

```diff
@@ -7,419 +7,521 @@ angular
+                                        content: "Click here to add a new project.",
+                                        reflex: true,
+                                        onShown: function (tour) {
+                                            $("#tour-next-button").prop("disabled", true);
+                                        }
+                                    },
+                                    {
+                                        path: "/discanno/#/projects",
+                                        element: "#pe_createProjectButton",
+                                        title: "Create a new project",
+                                        delay: 1,
+                                        placement: "left",
+                                        content: "Enter a name for the project, and select the scheme we created earlier.",
+                                        reflex: true,
+                                        onShown: function (tour) {
+                                            $("#tour-next-button").prop("disabled", true);
+                                        }
+                                    },
+                                    {
+                                        path: "/discanno/#/projects",
+                                        element: "#pe_edit_project",
+                                        title: "Edit project",
+                                        placement: "right",
+                                        content: "Click on the edit icon to add annotators to your project. Add the user that you created in the other tutorial. Then click 'Next'.",
+                                    },
+                                    {
+                                        path: "/discanno/#/projects",
+                                        element: "#pe_expand_documents",
+                                        title: "Show documents",
+                                        placement: "right",
+                                        content: "Click on the 'plus' if you want to see or edit the set of documents of a project. Click on '+Document' and upload some '.txt' file.",
+                                    },
+                                    {
+                                        element: "#1-projects-header",
+                                        title: "Well, that's it.",
+                                        content: "Your annotator can now log in and start working.",
+                                        path: "/discanno/#/projects",
+                                    },
+                                    {
+                                        element: "#pe_export",
+                                        title: "Ah, not quite.",
+                                        placement: "left",
+                                        content: "When your annotators are done, you can export your data as XML or XMI files!",
```

Comment:
We should stick to one notation: either "XML"/"XMI" or "xml"/"xmi" I think upper case is correct. Currently it is different on the projects page.